### PR TITLE
add study for BraveAdblockDefault1pBlocking

### PR DESCRIPTION
**Pull request by antonok-edm**
Applying https://github.com/brave/brave-variations/pull/109 on production, as it appears to work correctly on staging.

### CODE REVIEW

**Review by pes10k — APPROVED** [review]
lgtm

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. @antonok-edm also mentioned that this was checked on `main` as per https://github.com/brave/brave-variations/pull/110/#issue-724637238.

**Pull request merged**
